hbase-hadoop-compat/src/main/java/org/apache/hadoop/hbase/metrics/ExceptionTrackingSource.java

```diff
@@ -62,4 +63,5 @@
   void quotaExceededException();
   void rpcThrottlingException();
   void callDroppedException();
+  void callTimedOut();
```

Comment:
I added this in this PR because it's a type of call drop that can happen a lot under excess load. I did not roll it into callDroppedException, because it does not manifest as a CallDroppedException on the client side.